modules/special-rules/pages/heavy-barrage.adoc

```diff
@@ -1,18 +1,20 @@
 = Heavy Barrage
 
 These weapons are like {barrage} weapons, but even more destructive.
-They are also cumbersome and slow to reload -- only war engines are big enough to maintain a constant feed of ammunition to such a weapon during a battle. 
+They are also cumbersome and slow to reload -- only war engines are big enough to maintain a constant feed of ammunition to such a weapon during a battle.
 
 ---
 
 When you shoot with a {heavy-barrage} weapon, proceed exactly as for a {barrage} weapon but with these differences:
 
-* The Firepower value is equal to *twice* the number of units under the Barrage template.
+* The Firepower value is equal to *twice* the number of units covered by the Barrage template.
 * If the weapon is on a War Engine unit then you may continue to move and shoot with this unit each turn as normal.
 Otherwise, you may not *move* or *shoot* again with the unit until it has had a full turn to reload.
 Turn the model around, mark it with a token or make a note so that you don't forget that it must reload before you can use it again.
 +
 +[TODO: Clarify what should happen if the unit is forced to move, as in a withdrawal. Presumably they still can't move and so they are automatically destroyed if the enemy gets close enough.]+
++
++[REPLY IJW: I think it would make more sense for the reloading to be interrupted than for the unit to die.]+
```

Comment:
I think that one could call it either way on this, and I have no strong preference -- your suggestion is less harsh of course, but marginally more complex because you must remember that it'll take **another** turn to reload, and it breaks the 'may not move' stipulation. I do think it would be good to define it one way or another rather than leave it as a gap.

Overall, I'm happy to go with either of the approaches that we've annotated here. Another one for a quick poll perhaps (which might even turn up another alternative) -- or do you think that we should just decide?

Reply:
Yes, a poll for some feedback sounds good.

Reply:
I tidied up a bit to merge both of the options we've suggested into one TODO note -- so this conversation has gone outdated. I'll resolve this conversation, and I think that we can tackle the TODO in this PR or at a later time.